integreat_cms/summ_ai_api/summ_ai_api_client.py

```diff
@@ -122,27 +128,36 @@ async def translate_text_fields(self, loop, text_fields):
         :returns: The list of completed text fields
         :rtype: list [ ~integreat_cms.summ_ai_api.utils.TextField ]
         """
+
         # Set a custom SUMM.AI timeout
         timeout = aiohttp.ClientTimeout(total=60 * settings.SUMM_AI_TIMEOUT)
         async with aiohttp.ClientSession(timeout=timeout) as session:
             # Create tasks for each text field
             tasks = [
-                loop.create_task(self.translate_text_field(session, text_field))
+                lambda: self.translate_text_field(session, text_field)
```

Comment:
```suggestion
                self.translate_text_field(session, text_field)
```
There is no such thing like an async lambda function, adding support for them was rejected by the Python core developers: https://bugs.python.org/issue33447